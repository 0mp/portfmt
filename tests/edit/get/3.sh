${PORTEDIT} get '(^PORT|CONFIG)' 3.in | diff -L 3.expected -L 3.actual -u 3.expected -
