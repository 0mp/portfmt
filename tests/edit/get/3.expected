sndio
1.5.0
yes
--prefix=${PREFIX}
--mandir=${PREFIX}/man
yes
